### DISCUSSION TITLE
Livewire 3: Are model properties locked by default in a form object?

### DISCUSSION THREAD

**Discussion by IamDMC** [livewire/livewire#8277] [1. Help] [answered]
The documentation says that componenet model properties are locked by default. Is this the same for model properties in form objects?

https://livewire.laravel.com/docs/properties#eloquent-models-and-locking

I don't know how to work with dev tool console to manipulate this property and test if it can be changed from the front end.

**Comment by PerryvanderMeer** [comment, marked as answer]
Model properties are locked via the `ModelSynth`, which will also work within forms.

https://github.com/livewire/livewire/blob/cac2f7b28f37980653aaec99d9de55de21b2ebaa/src/Features/SupportModels/ModelSynth.php#L70-L80

**Reply by IamDMC** [reply]
Thank you!
